packages/@aws-cdk/aws-opensearchservice/package.json

```diff
@@ -89,11 +89,25 @@
     "@aws-cdk/integ-runner": "0.0.0",
     "@aws-cdk/cfn2ts": "0.0.0",
     "@aws-cdk/pkglint": "0.0.0",
+    "@aws-cdk/aws-certificatemanager": "0.0.0",
```

Comment:
Why are all these dependencies now necessary? I'm not seeing a change that would have necessitated it.

Reply:
Each of these was causing a build warning -- they're not strictly necessary, but it also seemed odd for the build to produce so many warnings.